lib/Auth/Process/WebAuthn.php

```diff
@@ -165,7 +189,12 @@ public function process(&$state)
         $state['webauthn:store'] = $this->store;
         Logger::debug('webauthn: userid: ' . $state['Attributes'][$this->usernameAttrib][0]);
 
-        if ($this->store->is2FAEnabled($state['Attributes'][$this->usernameAttrib][0], $this->defaultEnabled) === false) {
+        $local_toggle = null;
+        if (array_key_exists($this->toggle, $state['Attributes']) && array_key_exists(0, $state['Attributes'][$this->toggle])) {
+            $local_toggle = $state['Attributes'][$this->toggle][0] === 'true';
```

Comment:
I would suggest `!empty($state['Attributes'][$this->toggleAttrib][0])` instead, so that users can supply both booleans or (non)empty strings.